include/oneapi/dpl/pstl/hetero/dpcpp/unseq_backend_sycl.h

```diff
@@ -148,9 +190,26 @@ struct reduce
 {
     _BinaryOperation1 __bin_op1;
 
+    // Reduce on local memory with subgroups
     template <typename _NDItemId, typename _GlobalIdx, typename _Size, typename _AccLocal>
     _Tp
-    operator()(const _NDItemId __item_id, const _GlobalIdx __global_idx, const _Size __n, _AccLocal& __local_mem) const
+    reduce_impl(const _NDItemId __item_id, const _GlobalIdx __global_idx, const _Size __n, _AccLocal& __local_mem,
+                std::true_type /*has_known_identity*/) const
```

Comment:
It would look/read better as  `/*has_known_identity=*/std::true_type` (unless we consistently put the comment after the parameter in our code).